### PR TITLE
Fix readmes (again) and add linter action

### PR DESCRIPTION
**Pull request by dnaka91**

### CODE REVIEW

**Comment by dnaka91** [comment]
Hm, the main build fails although I didn't touch any related files.
Maybe you should commit a lock file for your dependencies?

File: .github/workflows/pr.yml [hunk 1]
```diff
@@ -4,47 +4,58 @@ on: [pull_request]
...
 jobs:
   build:
-
     runs-on: ubuntu-latest
-
     strategy:
       matrix:
         node-version: [12.x]
+    steps:
+      - uses: actions/checkout@v1
```
Comment (negue) [inline .github/workflows/pr.yml:12]: wonder why it worked without the 2 spaces :/

File: .markdownlint.yml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+default: true
```
Comment (negue) [inline .markdownlint.yml]: could you move the config files to `tools/configs` ?

File: .github/workflows/pr.yml (hunk 1 above)
Comment (dnaka91) [reply .github/workflows/pr.yml:12]: It's just the formatter. Both are valid yaml but the default setting of VS Code idents it.

File: .markdownlint.yml (hunk 2 above)
Comment (dnaka91) [reply .markdownlint.yml]: Others like `.editorconfig`, `.prettierrc`, `tslint.json` and many more are all tool configurations but aren't in the `tools/config` folder neither though 🤔😋

Comment (negue) [reply .markdownlint.yml]: There is a difference of daily used and cli tools (ng, npm,etc) 
and tools you may use once a week (and by npx something) ^^

I think all config files at root are mandatory right now, but if its moveable I'll try to move em too

Comment (dnaka91) [reply .markdownlint.yml]: True that. Both tools have an option to define the config location so I will move it :+1:

**Comment by negue** [comment]
TYVM :)

**Pull request merged**